template.yml

```diff
@@ -38,17 +38,32 @@ Resources:
         CleanScheduledEvent:
           Type: Schedule
           Properties:
-            Schedule: rate(1 hour)
+            Schedule: !Ref CleanSchedule
 
   LogGroup:
     Type: AWS::Logs::LogGroup
     Properties:
       LogGroupName: !Sub /aws/lambda/${Clean}
+      RetentionInDays: !Ref LogRetentionInDays
 
 Parameters:
+  CleanSchedule:
+    Type: String
+    Description: >-
+      How frequently to run the janitor, as an EventBridge schedule expression.
+      https://docs.aws.amazon.com/eventbridge/latest/userguide/scheduled-events.html
+    Default: rate(1 hour)
+
+  LogRetentionInDays:
+    Type: Number
+    Description: >-
+      How long to retain the execution logs for the janitor.
+    Default: 14
+    AllowedValues: [1, 3, 5, 7, 14, 30, 60, 90, 120, 150, 180, 365, 400, 545, 731, 1827, 3653]
+
   VersionsToKeep:
     Type: Number
     Description: >-
       How many versions to keep, even if they are not aliased.
     Default: 3
-    MinValue: 0 # don't keep anything except $Latest
+    MinValue: 0 # don't keep anything except $LATEST
```

Comment:
Why is this needed here ?

Reply:
Because `$LATEST` is how AWS names the latest version. They never call it `$Latest`.